src/wymeditor/editor/undo-redo.js

```diff
@@ -86,6 +86,12 @@ WYMeditor.UndoRedo.prototype._add = function () {
         return;
     }
 
+    if (wym.ih._isResizeHandleAttached()) {
+        // so that the resize handle
+        // does not end up as content in the history
+        wym.ih._detachResizeHandle();
```

Comment:
I really wish we had called this variable `imageHandler` instead of `ih`. I had to grep through source to figure out what was happening.